### DISCUSSION TITLE
2024/08/19/tailscale-451-error

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#36] [Blog comments]
# 2024/08/19/tailscale-451-error

Возникает при попытке установки Tailscale на хостах в некоторых странах.

https://blog.amd-nick.me/2024/08/19/tailscale-451-error

**Comment by CrazyHackGUT** [comment]
А там же вроде можно просто задать энвы `http_proxy`, `https_proxy`. Их и `curl` хавает, и `apt`. Потом ансетнуть и все.

**Reply by AMD-NICK** [reply]
Расскажи подробнее, пожалуйста

**Comment by MikleBas** [comment]
Привет! а без сокса никак не обойтись?
что за русофобы такие эти авторы??!!

**Reply by AMD-NICK** [reply]
Привет, других способов не знаю, сорри. Если снова столкнусь и найду решение без проксей, то обновлю пост и постараюсь не забыть пингануть в комментах

**Reply by MikleBas** [reply]
За инфу которую ты написал Спасибо!
PS а с остальным разобрался, все работает :)
